pandas/tests/frame/test_missing.py

```diff
@@ -753,17 +751,20 @@ def test_rowwise_alt(self):
                         1: [1, 2, 3, 4, 3, 2, 1, 0, -1]})
         df.interpolate(axis=0)
 
-    def test_interp_leading_nans(self):
+    @pytest.mark.parametrize("check_scipy", [
+        False, pytest.param(True, marks=td.skip_if_no_scipy)
+    ])
+    def test_interp_leading_nans(self, check_scipy):
         df = DataFrame({"A": [np.nan, np.nan, .5, .25, 0],
                         "B": [np.nan, -3, -3.5, np.nan, -4]})
         result = df.interpolate()
         expected = df.copy()
         expected['B'].loc[3] = -3.75
         assert_frame_equal(result, expected)
 
-        tm._skip_if_no_scipy()
```

Comment:
I was thinking of using ``kwargs`` for this function as noted in the subsequent comment, but it would slightly change the test given ``expected`` is a copy of the interpolation being done without ``scipy``. The subsequent ``scipy`` interpolation is still compared back to that object for equality, which may or may not be intentional